lib/active_merchant/billing/network_tokenization_credit_card.rb

```diff
@@ -0,0 +1,16 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class NetworkTokenizationCreditCard < CreditCard
```

Comment:
We have the `PaymentToken` class that intends to abstractly represent tokenized data. If we have a better understanding on how to model this kind of tokenized data, then I believe we should extend `PaymentToken` rather than introduce a new `CreditCard` subclass.